src/components/settings/SettingsAccessTab/SettingsAccessSubsidyRequestManagement.jsx

```diff
@@ -36,6 +36,12 @@ const SettingsAccessSubsidyRequestManagement = ({
     }
   }, [disabled, subsidyRequestsEnabled]);
 
+  useEffect(() => {
```

Comment:
I think we want `if (!disabled && !subsidyRequestsEnabled)...` but keep the dependency array the same.

Reply:
for those watching at home, adding `!subsidyRequestsEnabled` will be prevent an API call if this is already enabled